book/src/storage-capacity-tracking.md

```diff
@@ -0,0 +1,42 @@
+# Storage Capacity Tracking
+
+## Status
+
+Status | Min K8s Version | Max K8s Version
+--|--|--
+Alpha | 1.19 | -
+
+## Overview
+
+Storage capacity tracking allows the Kubernetes scheduler to make more
+informed choices about where to start pods which depend on unbound
+volumes with late binding (aka "wait for first consumer"). Without
+storage capacity tracking, a node is chosen without knowing whether
+those volumes can be made available for the node. Volume creation is
+attempted and if that fails, the pod has to be rescheduled,
+potentially landing on the same node again. With storage capacity
+tracking, the scheduler filters our nodes which do not have enough
+capacity.
+
+> For design information, see the [enhancement
+> proposal](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/1472-storage-capacity-tracking).
+
+## Usage
+
+To support rescheduling of a pod, a CSI driver deployment must:
+- return the `ResourceExhausted` gRPC status code in `CreateVolume` if
+  capacity is exhausted
+- use external-provisioner >= 1.6.0 because older releases did not
+  properly support rescheduling after a `ResourceExhausted` error
+
+To support storage capacity tracking, a CSI driver deployment must:
```

Comment:
Also the driver needs to report capacity in the csi GetCapacity call.

Reply:
Sometimes one has to point out the obvious :joy:  Added.